### ISSUE TITLE
Bots are moved to spectate and causes rounds to end constantly

### ISSUE THREAD

**Issue by Fla5hFyre** [splewis/csgo-retakes#259]
Trying to be able to play retakes with bots but it doesn't seem to work,

### Expected behavior
Bots are added to the teams as usual when doing bot_add

### Actual behavior
Bots are moved to spectate automatically and in the following round the game constantly kicks and adds the bots and thus starts a new round constantly until bot_kick is typed.

### Steps to reproduce
- Plugin version: 0.3.4
- Sourcemod version: 1.10.0.6488
- Steps to reproduce (please be specific): Add a bot to the server when retakes is active

**Comment by splewis** [comment]
The plugin isn't designed to work with bots.

**Comment by Fla5hFyre** [comment]
What difference do bots make? All you have to do is let them spawn like a player and they will work normally.

**Comment by Fla5hFyre** [comment]
It would be very helpful to simply allow bots to spawn as it's the best way to test out all the spawns and timings and stuff if you are making your own.

**Comment by splewis** [comment]
> What difference do bots make?

Not much. The plugin could work with bots, but somebody would need to send a pull request to make the needed changes. There's lots of `IsFakeClient` calls that would need to be adjusted to remove gotv clients and such.

**Comment by Fla5hFyre** [comment]
Ah gotcha, I see what you mean. Anyway this would be really nice if someone could implement it in the future.

**Comment by borzaka** [comment]
You can test your spawns as mentions here: https://github.com/splewis/csgo-retakes#creating-and-editing-spawns

> use `sm_goto` to go to a spawn. Spawns are indexed from 0 upwards. Example: `!goto 10` to go to spawn 10.

> use `sm_iteratespawns` to fake `sm_goto` commands every few seconds to visit every spawn in the map.

But you can't test the timings, but I'am not sure that the BOTs would run straight to the bombsite.

**Comment by Fla5hFyre** [comment]
Yeah I know you can see where the spawns are but it would just be nice to see the gamemode working fully with bots. Bots should work fine, as long as the bomb is planted, so it might work better with the auto plant plugin, but I think it should still work fine regardless, the bots just might be a bit lost until the one on site plants the bomb.
